HousingSearchApi/V2/Gateways/SearchGateway.cs

```diff
@@ -17,22 +18,49 @@ public SearchGateway(IElasticClient elasticClient)
     public async Task<SearchResponseDto> Search(string indexName, SearchParametersDto searchParams)
     {
         var searchResponse = await _elasticClient.SearchAsync<object>(s => s
-                .Index(indexName)
-                .Query(q => q
-                    .SimpleQueryString(qs => qs
-                        .Fields("*")
-                        .Query(searchParams.SearchText)
+            .Index(indexName)
+            .Query(q => q
+                .Bool(b => b
+                    .Should(
+                        MultiMatchSingleField(searchParams.SearchText, boost: 6),
+                        MultiMatchAcrossFields(searchParams.SearchText, boost: 2)
                     )
                 )
-                .Size(searchParams.PageSize)
-                .From((searchParams.PageNumber - 1) * searchParams.PageSize)
-                .TrackTotalHits(true)
+            )
+            .MinScore(15)
```

Comment:
Seems very arbitrary w/o knowing what values we're dealing with & what this number means.
Perhaps, a constant with a comment could be useful explaining the scale? Like 2-6 = bad results like ...

Reply:
Min score < 15 gives a bunch of coincidental matches .. like couple character matches of house numbers, etc but nothing else in the search text